pkg/proxy/apis/config/fuzzer/fuzzer.go

```diff
@@ -51,6 +51,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 			if obj.Logging.Format == "" {
 				obj.Logging.Format = "text"
 			}
+			obj.Windows = kubeproxyconfig.KubeProxyWindowsConfiguration{}
```

Comment:
You don't need to do that, and in fact you don't _want_ to do that; the `c.FuzzNoCustom(obj)` will already have fuzzed all of the scalar-typed fields (including `obj.Windows.RunAsService`), and this line will effectively undo that and force `RunAsService` to always be `false`.

Or did you add this to avoid unit test problems? It's possible that we're just going to have to add the new fields to the `v1alpha` config too...

Reply:
Yes, the unit test fails if I don't add that.
Should I simply add `WindowsRunAsService` to `v1alpha1.KubeProxyConfiguration`?